testcases/crypto/ec_func.c

```diff
@@ -1887,6 +1935,191 @@ CK_RV run_TransferECCKeyPairSignVerify()
     return rc;
 }
 
+/**
+ * Tests the EP11 token protected key option. PKEY_MODE ENABLE4NONEXTR must be
```

Comment:
What is the default value of PKEY_MODE?  Unfortunately there is the option "DEFAULT" which I guess is not the default value if PKEY_MODE is not specified in the config file.  Since the default config file does not specify PKEY_MODE (except for the documentation of it), I am just wondering if we will ever execute this test case in the CI or not?

Reply:
You are right: the default for PKEY_MODE is in fact DEFAULT, so the protected key option will not be enabled in CI automatically. This is a general problem with all of our token options. Many of the internal tests depend on various token options and will always be executed using defaults.